doc/api/api_changes.rst

```diff
@@ -39,7 +39,7 @@ API Changes for 3.0.0
 Drop support for python 2
 -------------------------
 
-Matplotlib 3 only supports python 3.5 and higher
+Matplotlib 3 only supports python 3.5 and higher.
```

Comment:
added on some minor grammar changes to your PR, but I think this sentence doesn't need the . (and Python should be capitalized).